inc/spbc-settings.php

```diff
@@ -5965,6 +5982,11 @@ function spbc__get_accordion_tab_info_block_html($for)
         $domain = 'https://l.cleantalk.org'
     );
 
+    $company_name = 'ClaenTalk';
```

Comment:
First one - there is an typo. Second one - please, get company name by default from [State](https://github.com/CleanTalk/security-malware-firewall/blob/master/lib/CleantalkSP/SpbctWP/State.php#L211)